client/src/pages/FloorPlan.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 
-const Desk = ({ desk }) => {
+const Desk = ({ desk, bookingForThisDesk }) => {
```

Comment:
bookingForThisDesk is a great name! (I would have just used booking :D )

Reply:
let's use greater than great than great